renderer/brave_content_settings_observer_browsertest.cc

```diff
@@ -478,8 +478,7 @@ IN_PROC_BROWSER_TEST_F(BraveContentSettingsObserverBrowserTest,
       contents()).c_str(), "");
   ASSERT_TRUE(NavigateIframeToURL(contents(), kIframeID, iframe_url()));
   ASSERT_EQ(child_frame()->GetLastCommittedURL(), iframe_url());
-  EXPECT_STREQ(ExecScriptGetStr(kReferrerScript,
-      child_frame()).c_str(), iframe_url().GetOrigin().spec().c_str());
+  EXPECT_STREQ(ExecScriptGetStr(kReferrerScript, child_frame()).c_str(), "");
```

Comment:
these tests only check document.referrer, but we should also ensure the request referrer is set correctly